src/zimscraperlib/zim/items.py

```diff
@@ -106,7 +104,7 @@ def download_for_size(url, on_disk, tmp_dir=None):
         size, _ = stream_file(url.geturl(), fpath=fpath, byte_stream=stream)
         return fpath or stream, size
 
-    def __init__(self, url: str, **kwargs):
+    def __init__(self, url: str, **kwargs: Any):
```

Comment:
I think that all *expected* (for which we have processing in the class) values should be individual optionnal keyword arguments with they expected type. We can leave `Any` for the rest

Reply:
Same logic would apply to classes inheriting `Item`

Reply:
I can only agree :)